documentation/api/development.md

```diff
@@ -25,13 +25,13 @@ A very simple [docker-compose.yml](../../docker-compose.yml) has been included t
 
 ## Run the application
 
-1. In your terminal, `cd` to the `api` directory of this repo.
+1. Run `cp frontend/.env.development frontend/.env.local` to create your local copy of the frontend environment variables
 2. Make sure you have [Docker Desktop](https://www.docker.com/products/docker-desktop/) installed & running.
-3. Run `make setup-local` to install dependencies
-4. Run `make init start` to build the image and start the container.
+3. Run `make -C api setup-local` to install dependencies
```

Comment:
As someone who has multiple tabs open in terminal for each project subdirectory, I should really get in the habit of using `-C` in my make commands instead 🤦 . thanks for adding this!